src/managed/OpenLiveWriter.ApplicationFramework/MenuDefinitionEntryCollection.cs

```diff
@@ -12,233 +13,28 @@ namespace OpenLiveWriter.ApplicationFramework
-        {
-            return new MenuDefinitionEntryEnumerator(this);
-        }
-
-        /// <summary>
-        /// Removes the first occurrence of a specific Command from the CommandCollection.
-        /// </summary>
-        /// <param name="value">The Command to remove.</param>
-        public void Remove(MenuDefinitionEntry value)
-        {
-            List.Remove(value);
-        }
-
-        /// <summary>
-        /// Supports a simple iteration over a MenuDefinitionEntryCollection.
-        /// </summary>
-        public class MenuDefinitionEntryEnumerator : object, IEnumerator
-        {
-            /// <summary>
-            /// Private data.
-            /// </summary>
-            private IEnumerator baseEnumerator;
-            private IEnumerable temp;
-
-            /// <summary>
-            /// Initializes a new instance of the MenuDefinitionEntryEnumerator class.
-            /// </summary>
-            /// <param name="mappings">The MenuDefinitionEntryCollection to enumerate.</param>
-            public MenuDefinitionEntryEnumerator(MenuDefinitionEntryCollection mappings)
-            {
-                temp = (IEnumerable)mappings;
-                baseEnumerator = temp.GetEnumerator();
-            }
-
-            /// <summary>
-            /// Gets the current element in the collection.
-            /// </summary>
-            public MenuDefinitionEntry Current
-            {
-                get
-                {
-                    return (MenuDefinitionEntry)baseEnumerator.Current;
-                }
-            }
-
-            /// <summary>
-            /// Gets the current element in the collection.
-            /// </summary>
-            object IEnumerator.Current
-            {
-                get
-                {
-                    return baseEnumerator.Current;
-                }
-            }
-
-            /// <summary>
-            /// Advances the enumerator to the next element of the collection.
-            /// </summary>
-            public bool MoveNext()
-            {
-                return baseEnumerator.MoveNext();
-            }
-
-            /// <summary>
-            /// Advances the enumerator to the next element of the collection.
-            /// </summary>
-            bool IEnumerator.MoveNext()
-            {
-                return baseEnumerator.MoveNext();
-            }
-
-            /// <summary>
-            /// Sets the enumerator to its initial position, which is before the first element in the collection.
-            /// </summary>
-            public void Reset()
-            {
-                baseEnumerator.Reset();
-            }
-
-            /// <summary>
-            /// Sets the enumerator to its initial position, which is before the first element in the collection.
-            /// </summary>
-            void IEnumerator.Reset()
-            {
-                baseEnumerator.Reset();
-            }
+            Add(mde);
         }
+        
```

Comment:
We might want to support StyleCop at some point, and this empty line violates (http://stylecop.com/docs/SA1508.html)[http://stylecop.com/docs/SA1508.html]